summarizer/datasets/correlation.ipynb

```diff
@@ -0,0 +1,432 @@
+    "    machine_scores = np.random.uniform(0, 1, (n_frames,))\n",
+    "    corrs.append(evaluate_scores(machine_scores, user_summary))\n",
+    "np.mean(corrs)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "execute_result",
+     "data": {
+      "text/plain": "0.22560639713572397"
+     },
+     "metadata": {},
+     "execution_count": 14
+    }
+   ],
+   "source": [
+    "# Human scores: should be maximum\n",
+    "corrs = []\n",
+    "for i in range(n_users):\n",
+    "    machine_scores = user_summary[i] # here we don't have scores from user but only summary\n",
+    "    corrs.append(evaluate_scores(machine_scores, np.delete(user_summary, i, 0)))\n",
+    "np.mean(corrs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using scores `[0.8 0.3 ... 0.1]` as target"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "execute_result",
+     "data": {
+      "text/plain": "0.0009742716877428895"
+     },
+     "metadata": {},
+     "execution_count": 15
+    }
+   ],
+   "source": [
+    "# Random scores: should be 0\n",
+    "corrs = []\n",
+    "for _ in range(50):\n",
+    "    machine_scores = np.random.uniform(0, 1, (n_frames,))\n",
+    "    corrs.append(evaluate_scores(machine_scores, user_scores))\n",
+    "np.mean(corrs)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "execute_result",
+     "data": {
+      "text/plain": "0.3319613038488442"
+     },
+     "metadata": {},
+     "execution_count": 16
+    }
+   ],
+   "source": [
+    "# Human scores: should be maximum\n",
+    "corrs = []\n",
+    "for i in range(n_users):\n",
+    "    machine_scores = user_summary[i] # here we don't have scores from user but only summary\n",
+    "    user_scores_ = np.expand_dims(np.delete(user_summary, i, 0).mean(axis=0), axis=0)\n",
+    "    corrs.append(evaluate_scores(machine_scores, user_scores_))\n",
+    "np.mean(corrs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "_Observations:_\n",
+    "* Using `user_summary` is consistent: random scores leads to 0 and human is high (~0.22)\n",
+    "* Using `user_scores` matches the paper: random also near 0 and human high (us ~0.33)\n",
+    "\n",
+    "In the paper they did not experiment this with SumMe since there is no importance score annotations.\n",
+    "\n",
+    "Here, we should use `user_scores` (in other words `/gtscore`) to be consistent with TVSum, even if `/gtscore` is computed from summaries of annotators instead of absolute frame importance score. But, these summaries were made by annotators directly, without usiing KTS. So it is not biased by KTS. Which is the dark side of using F-score. So we can assume were're fine?"
```

Comment:
```suggestion
    "Here, we should use `user_scores` (in other words `/gtscore`) to be consistent with TVSum, even if `/gtscore` is computed from summaries of annotators instead of absolute frame importance score. But, these summaries were made by annotators directly, without usiing KTS. So it is not biased by KTS. Which is the dark side of using F-score. So we can assume we're fine?"
```